packages/_vue3-migration-test/src/x-modules/facets/x-module.ts

```diff
@@ -0,0 +1,38 @@
+import { Facet } from '@empathyco/x-types';
+import { createSimpleFacetStub } from '../../../../x-components/src/__stubs__/facets-stubs.factory';
```

Comment:
![Screenshot 2024-05-07 at 5 58 10 PM](https://github.com/empathyco/x/assets/10746604/7b7f2e8a-40e8-4209-8690-65ce9c8ecaca)

You can remove `rootDir` from `tsconfig` to point to the monorepo instead of `src`